img/phone/phone.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M23.9911 19.7531C23.9479 19.3576 23.7502 19.0043 23.4343 18.7596L18.8089 15.1847C18.0226 14.578 16.9618 14.506 16.1035 15.0023L15.2529 15.4938C14.7345 15.7924 14.0932 15.753 13.6189 15.3863C12.9316 14.8573 11.9053 14.0203 10.9232 13.0373C9.94874 12.0619 9.12692 11.055 8.60947 10.382C8.24659 9.91072 8.20627 9.27042 8.50578 8.75109L8.9973 7.89863C9.49365 7.03853 9.42258 5.97584 8.81583 5.19252L5.23972 0.567448C4.99596 0.251654 4.6446 0.0539375 4.24811 0.0097816C3.85161 -0.0362961 3.46473 0.0817809 3.15847 0.335185C2.37317 0.983132 1.36416 1.82983 1.01379 2.18115C0.303349 2.89154 0 3.82176 0 5.29433C0 7.01167 3.15275 13.3992 6.94966 17.0374C10.6006 20.8484 16.9887 24 18.7081 24C20.1798 24 21.1101 23.6967 21.8186 22.9863C22.1978 22.6081 23.1386 21.4792 23.6647 20.8418C23.9191 20.5345 24.0343 20.1486 23.9911 19.7531Z" fill="#00343D"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M16.3305 1.97177C18.984 2.94054 21.0599 5.01647 22.0277 7.67004C22.1019 7.87361 22.2944 8 22.499 8C22.5561 8 22.6133 7.98996 22.6695 7.9689C22.9303 7.87366 23.0646 7.58679 22.9694 7.32608C21.9013 4.3947 19.6068 2.09919 16.6735 1.03013C16.4167 0.935867 16.1259 1.07024 16.0306 1.329C15.9353 1.58868 16.0698 1.87648 16.3305 1.97177Z" fill="#00343D"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.2883 5.84564C16.6071 6.32228 17.6777 7.39406 18.1534 8.71245C18.2187 8.89022 18.3851 9 18.5637 9C18.6133 9 18.663 8.99217 18.7117 8.97473C18.9383 8.89282 19.0558 8.64361 18.9739 8.41703C18.4103 6.85815 17.1437 5.5903 15.5844 5.02652C15.3596 4.94372 15.1088 5.06137 15.026 5.28795C14.9442 5.51452 15.0618 5.76373 15.2883 5.84564Z" fill="#00343D"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M14.3021 8.88217C14.6828 9.0185 14.9809 9.31658 15.1172 9.69828C15.1845 9.88457 15.359 10 15.5453 10C15.5953 10 15.6471 9.99183 15.6979 9.97275C15.9351 9.88821 16.0578 9.62829 15.9733 9.39203C15.7452 8.75312 15.2472 8.25512 14.6074 8.02697C14.372 7.94154 14.1121 8.06607 14.0267 8.30145C13.9422 8.53771 14.0649 8.79763 14.3021 8.88217Z" fill="#00343D"/>
+</svg>
```

Comment:
These SVGs are not optimized, you can use these app
For SVG: https://imageoptim.com/mac
For PNG, JPEG and among others: https://tinypng.com/
@Jefer-Max

Reply:
Done

Reply:
done